.github/workflows/android-ci.yml

```
@@ -1,49 +1,43 @@
+    - name: Generate local.properties
+      run: |
+        echo "${{ secrets.LOCAL_PROPERTIES }}" > ./local.properties
+
+    # google-services.json 생성 (필수, 제대로 동작하게 수정)
+    - name: Generate google-services.json
+      run: |
+        echo "${{ secrets.GOOGLE_SERVICES }}" > ./app/google-services.json
+
+    # 캐시(선택)
+    - name: Gradle Caches
+      uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)